include/deal.II/grid/grid_tools.h

```diff
@@ -587,6 +636,35 @@ namespace GridTools
                        const std::vector<bool>       &marked_vertices = std::vector<bool>());
 
   /**
+   * Find and return the number of the used vertex (or marked vertex) in a
+   * given mesh that is located closest to a given point. Use the given
+   * mapping to compute the actual location of the vertices.
```

Comment:
Can you add the same example here about how the mapping is usually the identity operation on vertices, except for example for Eulerian mappings?

Reply:
done.